src/commons/sagas/BackendSaga.ts

```diff
@@ -466,6 +484,7 @@ function* BackendSaga(): SagaIterator {
         gradedFilter,
         pageParams,
         filterParams
+        // sortedBy
```

Comment:
Is this going to be used/removed?

Reply:
It will be used for backend column sorting onced implemented, commented for now in case it causes errors.

Reply:
This has been uncommented in the latest commit and the various issues that comes with it has been fixed.